modules/demo/rapids-api-server/package.json

```diff
@@ -8,7 +8,7 @@
   },
   "scripts": {
     "test": "tap \"test/**/*.test.js\"",
-    "build": "tsc -p ./tsconfig.json",
+    "build": "tsc -p ../../../tsconfig.json",
```

Comment:
```suggestion
    "build": "tsc -p /tsconfig.json",
```
?

Reply:
There's no TypeScript in this project presently...